### PR TITLE
Add order-book imbalance indicators

### LINKED ISSUE #1268: [Indicators] OBI(Order‑Book Imbalance) 지표 구현 계획
Title: [Indicators] OBI(Order‑Book Imbalance) 지표 구현 계획

Summary
- #1264 아이디어를 실행 가능한 형태로 구현합니다: 원시 오더북 스냅샷을 입력으로 받아 상단 L‑레벨 잔량 합산 후 `(bid-ask)/(bid+ask+ε)`를 산출하는 OBI 지표 노드를 `qmtl.runtime.indicators`에 추가합니다. 선택적으로 EMA 스무딩 조합 API를 제공합니다.

Rationale
- 현재 `transforms/order_book_imbalance.py`는 이미 산출된 bid/ask 볼륨을 입력으로 받는 저수준 변환입니다. 전략에서 바로 쓸 수 있는 고수준 지표(스냅샷→레벨 합산→OBI)를 제공해 결합 비용을 낮춥니다.

API Proposal
- `order_book_obi(source: Node, *, levels: int = 1, epsilon: float = 1e-9, name: str | None = None) -> Node`
  - 입력: `source`는 최신 스냅샷에 `{"bids": [(px,size)|size,...], "asks": [...]} \n` 구조를 내보냅니다.
  - 처리: 양측 상위 `levels` 크기 합산 후 `(bid-ask)/(bid+ask+ε)` 계산.
  - 출력: `float` 범위 [-1, 1]. 비어 있는 장부 또는 합계=0이면 0.0을 반환(이벤트 연속성 유지).
- `order_book_obi_ema(source: Node, *, levels: int = 1, ema_period: int = 20, epsilon: float = 1e-9, name: str | None = None) -> Node`
  - 구현: `order_book_obi(...)`를 구성한 뒤, 기존 `indicators.ema`로 스무딩한 노드를 래핑하여 반환.

Design Notes
- 레벨 합산은 내부에서 처리하여 `transforms/depth_node` 의존 없이도 동작. (원하면 후속 최적화에서 재사용 검토)
- 입력 레벨이 `(price,size)` 또는 단일 `size` 모두 허용. 비정상 값은 `float(...)` 변환 실패 시 건너뜀.
- 반환 정책: `total==0` 시 `0.0`. 스냅샷 부재 시 `None`.

Acceptance Criteria
- 단위 테스트(빠른):
  - 상단 1/3 레벨 케이스에서 기대 OBI 산출(정/부/균형 예시).
  - `(price,size)`/`size` 두 형태 모두 지원 검증.
  - 합계=0 케이스에서 0.0 반환, 스냅샷 미존재 시 `None`.
  - `order_book_obi_ema(..., ema_period=3)`가 수작업 EMA와 일치.
- 문서/사용 예: `qmtl/runtime/indicators/README.md`에 간단한 예제/주의사항 추가.
- 내보내기: `qmtl/runtime/indicators/__init__.py`에 심볼 추가.

Out of Scope
- 멀티‑레벨 OBI_L 및 심도 경사(Refs #1265).
- OBI 상태머신/레짐 전환(Refs #1266).
- Mid‑price t+Δ 예측 실험 스캐폴드(Refs #1267).

Work Plan / Tasks
- [ ] 지표 구현: `qmtl/runtime/indicators/order_book_obi.py`
- [ ] 스무딩 조합: `order_book_obi_ema` (내부에서 `ema` 재사용)
- [ ] Export: `qmtl/runtime/indicators/__init__.py` 업데이트
- [ ] README 사용 예 추가: `qmtl/runtime/indicators/README.md`
- [ ] 테스트 추가: `tests/runtime/indicators/test_obi.py`
- [ ] Preflight: `PYTHONFAULTHANDLER=1 uv run --with pytest-timeout -m pytest -q --timeout=60 --timeout-method=thread --maxfail=1 -k 'obi'`
- [ ] Full: `uv run -m pytest -W error -n auto -k 'obi'`

Notes (Dev Guidelines)
- 의존성/빌드는 `uv` 사용. 문서 수정 시 `uv run mkdocs build`로 검증.

References
- Refs #1264

### PR DESCRIPTION
**Pull request by hyophyop**
## Summary
- add an order_book_obi indicator that converts raw order-book snapshots into normalized imbalance values with optional EMA smoothing
- expose the new indicators through qmtl.runtime.indicators and document usage examples
- cover the indicator behaviour with dedicated unit tests across level formats, empty books, and EMA smoothing

## Testing
- PYTHONFAULTHANDLER=1 uv run --with pytest-timeout -m pytest -q --timeout=60 --timeout-method=thread --maxfail=1 -k 'obi'
- uv run -m pytest -W error -n auto -k 'obi'

Fixes #1268

------
https://chatgpt.com/codex/tasks/task_e_68df4c1057648329bbc5d52b099dd0c1

### CODE REVIEW

File: qmtl/runtime/indicators/order_book_obi.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+        denominator = bid_total + ask_total + epsilon
+        return (bid_total - ask_total) / denominator
+
+    return Node(
+        input=source,
+        compute_fn=compute,
+        name=name or "order_book_obi",
+        interval=source.interval,
+        period=1,
```
[the remark this answers is not in the document]
Comment (hyophyop) [reply qmtl/runtime/indicators/order_book_obi.py:92]: @codex fix comments
